api/common.js

```diff
@@ -413,6 +416,125 @@ exports.load_github_detail = function(service_name, cb) {
     }).catch(cb)
 }
 
+exports.mag_evaluate = function(query, projectid){
+    console.log(config.mag.subscription_key);
+    console.log(query);
```

Comment:
If you are done with debugging, please remove the debug log or set it to console.debug.